src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useState, useEffect } from 'react';
+
+import { Person } from '../types';
+
+import { getPeople } from '../api';
+
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setHasError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  if (isLoading) {
+    return <Loader />;
```

Comment:
Your Loader doesnt work as expected.  During loading your title " People Page " is hidden. You can fix it by using conditional rendering, amd your Loader will work properly as in workink version